tests/test_metrics.py

```diff
@@ -68,31 +68,31 @@ def test(self):
         self.assertTrue(ret["baz"]["metric_json"] == ["baz"])
 
         ret = self.dvc.metrics.show(
-            "metric_tsv", typ="tsv", xpath="0,0", all_branches=True
+            "metric_tsv", typ="TsV", xpath="0,0", all_branches=True
```

Comment:
Let's move case testing into a separate test. Consider adding it to `tests/unit/repo/metrics/`.